src/saveload/saveload.h

```diff
@@ -353,6 +353,9 @@ enum SaveLoadVersion : uint16 {
 	SLV_LINKGRAPH_SECONDS,                  ///< 308  PR#10610 Store linkgraph update intervals in seconds instead of days.
 	SLV_AI_START_DATE,                      ///< 309  PR#10653 Removal of individual AI start dates and added a generic one.
 
+	SLV_MTILE_DEPOTS_ADD_HANGARS,           ///< 310  PR#9577 Multi-tile depots. Add DepotIDs to airports that have a hangar.
```

Comment:
It's not necessary to mention multi-tile depots, as this change doesn't affect that yet.

Also, I've previously done only one savegame bump per PR, with multiple commits using it. Not sure if that's best practice, though, so maybe someone more experienced can chime in. 😛 

```suggestion
	SLV_HANGER_DEPOT_IDS,                   ///< 310  PR#10691 Add DepotIDs to airports that have a hangar.
```

Reply:
Ok. I will remove that mention. Anyway, I think it is ok to keep it, as some time from now maybe somebody reading the code won't know these changes are related to PR#9577. I will change this anyway.

I've been previously told each commit should work on its own. So I need one savegame bump per commit. I will keep it with two separated savebumps if I am not told otherwise.